### DISCUSSION TITLE
v4 stability for use in development (as of dev14)

### DISCUSSION THREAD

**Discussion by kennethjyang** [isar/isar#1412] [Help] [answered]
v4 looks really exciting but it is also _very different_ from v3. I just started working on a project (with v3) and I'm wondering if I should instead use the dev builds of v4 since the API has significant (good) changes. Is v4 stable enough to use in development? I understand the notes say not "production" ready, but I am far from production.

Thanks!

**Comment by simc** [comment, marked as answer]
v4 is definitely ready to be used in development. The API should be quite similar however. I migrated a big project within a few minutes

**Reply by kennethjyang** [reply]
Great! Yeah my project is heavily reliant on links and async functions, both of which have been removed or changed in v4 so I wanted to switch sooner. Thanks!

**Comment by neiljaywarner** [comment]
@simc  when will v4 be production ready esp if we do NOT need to migrate from v 3?
